docker/kafka/scripts/run-consumer.sh

```diff
@@ -8,5 +8,4 @@ fi
 
 "$KAFKA_BIN"/kafka-console-consumer.sh \
    --bootstrap-server "$KAFKA_BROKER" \
-   --topic sfx-employee \
-   --max-messages $((10 + RANDOM % 100))
```

Comment:
I think this relate to @atoulme 's slack message gist. 
At the core of the issues, one of the crucial ones was that the kafka tests would not detect some of kafka's metrics based on the activity of kafka itself. There were multiple things to it:
The message sender had a random number of messages to send. It would send 10 + at most 100.
The broker would only emit metrics related to its roll stats if it rolled a segment.

Reply:
beat me to it